src/dmclock_server.h

```diff
@@ -1301,13 +1310,15 @@ namespace crimson {
       // pull convenience constructor
       PullPriorityQueue(typename super::ClientInfoFunc _client_info_f,
 			AtLimitParam at_limit_param = AtLimit::Wait,
-			double _anticipation_timeout = 0.0) :
+			double _anticipation_timeout = 0.0,
+			Counter _erase_max = 100) :
 	PullPriorityQueue(_client_info_f,
-			  std::chrono::minutes(10),
-			  std::chrono::minutes(15),
-			  std::chrono::minutes(6),
+			  std::chrono::seconds(300),
+			  std::chrono::seconds(600),
+			  std::chrono::seconds(5),
```

Comment:
How did you arrive at 5 seconds? Did you do any profiling to see if this is good? Have you observed differences in behavior?

I'm thinking to your example of 75,000 rbd images. If you can only clean 100 at a time it will take 750 rounds to clean all of them. If those rounds are 1 minute apart, that's 750 minutes, which is 12.5 hours. If those rounds are 5 seconds apart then it's just a little over an hour. And that's a good argument.

But what about the case where no cleaning is to be done?

I can be convinced either way. Let me throw out an idea. What if there's a short check_time if you hit erase_max, say of 5 seconds. But if you didn't hit erase_max last time, you'll have a longer check_time.

I'd welcome your thoughts and feedback.

Reply:
@ivancich thank you. I'm try to fix as above and repush it once I'm ready.

Reply:
Great! Thank you!